app/web/src/features/auth/login/Login.tsx

```diff
@@ -59,62 +43,50 @@ export default function Login() {
   return (
     <>
       {authenticated && <Redirect to={redirectTo} />}
-      {/***** MOBILE ******/}
-      <Hidden mdUp>
-        <div className={authClasses.page}>
-          <AuthHeader>{LOGIN_HEADER}</AuthHeader>
-          {error && (
-            <Alert className={authClasses.errorMessage} severity="error">
-              {error}
-            </Alert>
-          )}
-          <LoginForm />
-          <Typography className={classes.signUp}>
-            {NO_ACCOUNT_YET + " "}
-            <Link className={classes.signUpLink} to={signupRoute}>
-              {SIGN_UP}
-            </Link>
-          </Typography>
-        </div>
-      </Hidden>
-
-      {/***** DESKTOP ******/}
-      <Hidden smDown>
-        <div className={authClasses.page}>
-          <header className={authClasses.header}>
-            <div className={authClasses.logoContainer}>
-              <CouchersLogo />
-              <div className={authClasses.logo}>{COUCHERS}</div>
-            </div>
-            <Typography className={classes.signUp}>
+      <div className={classNames(authClasses.page, authClasses.pageBackground)}>
+        <header className={authClasses.header}>
+          <div className={authClasses.logoContainer}>
+            <CouchersLogo />
+            <div className={authClasses.logo}>{COUCHERS}</div>
+          </div>
+        </header>
+        <div className={authClasses.content}>
+          <div className={authClasses.introduction}>
+            <Typography
+              classes={{ root: authClasses.title }}
+              variant="h1"
+              component="span"
+            >
+              {INTRODUCTION_TITLE}
+            </Typography>
+            <Typography
+              classes={{ root: authClasses.subtitle }}
+              variant="h2"
+              component="span"
+            >
+              {INTRODUCTION_SUBTITLE}
+              <Divider className={authClasses.underline}></Divider>
+            </Typography>
+          </div>
+          <div className={authClasses.formWrapper}>
```

Comment:
Now this doesn't seem to be centre aligned on mobile, whereas the sign up form does centre align correctly:
![Screenshot 2021-09-19 at 02 28 28](https://user-images.githubusercontent.com/13669362/133912584-2e4cd962-dea2-470a-b7ab-a0e47feb9984.png)